content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+API, Standard Channel features can continue to evolve with backward-compatible
+additions over time, and we certainly expect further refinements and
+improvements to these new features in the future. For more information on how
+all of this works, refer to the [Gateway API Versioning
+Policy](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+#### [HTTPRoute Timeouts](https://gateway-api.sigs.k8s.io/guides/http-routing/)
+
+[GEP-1742](https://gateway-api.sigs.k8s.io/geps/gep-1742/) introduced the
+`timeouts` stanza into HTTPRoute, permitting configuring basic timeouts for
+HTTP traffic. This is a simple but important feature for proper resilience
+when handling HTTP traffic, and it is now Standard.
+
+For example, this HTTPRoute configuration sets a timeout of 300ms for traffic
+to the `/face` path:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: HTTPRoute
+metadata:
+  name: face-with-timeouts
+  namespace: faces
+spec:
+  parentRefs:
+    - name: my-gateway
+      kind: Gateway
+      port: 80
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /face
+    backendRefs:
+    - name: face
+      port: 80
+    timeouts:
+      request: 300ms
+```
+
+For more information, check out the [HTTP routing
+documentation](https://gateway-api.sigs.k8s.io/guides/http-routing/). (Note
+that this applies only to HTTPRoute timeouts. GRPCRoute timeouts are not yet
+part of Gateway API.)
+
+#### [Gateway Infrastructure Labels and Annotations](https://gateway-api.sigs.k8s.io/guides/grpc-routing/)
+
+Gateway API for north-south traffic has always had the concept that declaring
+a Gateway causes the backing infrastructure (for example, a Service and a
+Deployment) to be created by the Gateway API controller. One complication with
+this approach is that sometimes it's important to label or annotate the
+infrastructure: for example, it's commonly necessary to use one of these
+mechanisms to tell a service mesh to incorporate the Gateway infrastructure.
+
+In 1.2.0, the Gateway `infrastructure` stanza moves to the Standard channel,
+allowing you to specify labels and annotations for the infrastructure created
+by the Gateway API controller. For example, the following Gateway
+configuration specifies both Linkerd and Istio injection, allowing the Gateway
+infrastructure to be incorporated into whichever service mesh you've
+installed:
+
+```
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: meshed-gateway
+  namespace: incoming
+spec:
+  gatewayClassName: meshed-gateway-class
+  listeners:
+  - name: http-listener
+    protocol: HTTP
+    port: 80
+  infrastructure:
+    labels:
+      istio-injection: enabled
+    annotations:
+      linkerd.io/inject: enabled
+```
+
+For more information, check out the [`infrastructure` API
+reference](https://gateway-api.sigs.k8s.io/reference/spec/#gateway.networking.k8s.io/v1.GatewayInfrastructure).
```

Comment:
I didn't realize we were missing docs for this - @keithmattix or @mikemorris any chance you can add something before this blog post goes out?

Reply:
I'm going to resolve this in favor of the newly-created https://github.com/kubernetes-sigs/gateway-api/issues/3437.

Reply:
https://github.com/kubernetes-sigs/gateway-api/pull/3455